docs/pre-converted-images.md

```diff
@@ -0,0 +1,35 @@
+# Trying pre-converted images
+
+We have several pre-converted stargz images on Docker Hub, mainly for benchmarking purpose. This doc lists these images in a table format. You can try them on your machine with our snapshotter. Please refer to README for the procedure.
+
+Please do not use them in production. You always can build your stargz images optimized for your workload, using `ctr-remote` command (Please also see README).
+
+## Pre-converted images
+
+This section contains a table of pre-converted images which can be used for benchmarking, testing, etc.
+
+In the following table, `Image Name` column contains placeholders `{{ suffix }}` for image names. One of the following suffixes comes there based on the type of image.
+
+- `org`: Legacy image copied from `docker.io/library` without optimization. Layers are normal tarballs.
+- `sgz`: Stargz-formatted version of the `org` images. Layers are stargz archives and we can see performance improvement on the pull operation.
+- `esgz`: Heavily optimized version of the `org` images, based on the workload. `ctr-remote images optimize` command is used for the optimization. We can see the performance improvement on run operation as well as pull.
+
+`Optimized Workload` column describes workloads used for building `esgz` images. We optimized these images for benchmarking which is based on [HelloBench](https://github.com/Tintri/hello-bench) so we specified "hello-world"-like workloads for the command. See [benchmarking script](/script/benchmark/hello-bench/src/hello.py) for the exact command option specified for `ctr-remote images optimize`. 
+
+|Image Name|Optimized Workload|
+---|---
+|`docker.io/ktokunaga/alpine:3.10.2-{{ suffix }}`|Executing `echo hello` on the shell|
```

Comment:
I'd suggest creating a dedicated Docker Hub org. Can be follow-up in future.

Reply:
I agree with this. Let's work on it on another thread.